jenkins/add-docker-build-job.sh

```diff
@@ -142,6 +154,27 @@ job_xml=${job_xml//'{insert-git-url}'/${git_url}}
 job_xml=${job_xml//'{insert-registry}'/${registry}}
 job_xml=${job_xml//'{insert-docker-credentials}'/${credentials_id}}
 job_xml=${job_xml//'{insert-container-repository}'/${repository}}
+
+
+if [ -n "${scm_poll_schedule}" ]
+then
+  scm_poll_ignore_commit_hooks_bool="false"
+  if [[ "${scm_poll_ignore_commit_hooks}" == "1" ]]
+  then
+    scm_poll_ignore_commit_hooks_bool="true"
+  fi
+  triggers_xml_node=$(cat <<EOF
+<triggers>
+  <hudson.triggers.SCMTrigger>
```

Comment:
will setting this trigger automatically poll the github URL that's set in the pipeline? 

also,  ```scm_poll_schedule``` is something like ```1 * * * * `` to have it poll every minute?